### ISSUE TITLE
The result is bad

### ISSUE THREAD

**Issue by duguiming111** [tensorflow/nmt#478]
![image](https://user-images.githubusercontent.com/32973161/85099548-3eccb980-b230-11ea-9776-f0a3bbbf305b.png)

I use the zh-en corpus of iwslt2015, and the Bleu value can only reach 5.3. I find that there are many such problems in the issues, which can't reach the result you give. Is there any problem in our operation?

**Comment by duguiming111** [comment]
When I run the code,
python3 nmt.py --attention=scaled_luong --src=zh --tgt=en --vocab_prefix=tmp/nmt_data_zh_en/vocab --train_prefix=tmp/nmt_data_zh_en/train --dev_prefix=tmp/nmt_data_zh_en/tst2012 --test_prefix=tmp/nmt_data_zh_en/tst2013 --out_dir=tmp/nmt_attention_model --num_train_steps=12000 --steps_per_stats=100 --num_layers=2 --num_units=512 --encoder_type=bi --dropout=0.2 --decay_scheme=luong234 --metrics=bleu
The best value of blue is 11.4，are there sone problems?

**Comment by maharajbrahma** [comment]
Bleu score depends on various factors like architecture used, hyper parameters, number of sentences in data set.

**Comment by watermelona** [comment]
@duguiming111   i am the begginer  of the nmt,   i can not find the zh-en corpus of iwslt2015,  can you tell me  how
 to get this dataset and  vocab ?
